src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -74,7 +77,7 @@ private static IEnumerable<string> GetSarifFiles(IEnumerable<string> files)
 				    directory = Path.GetDirectoryName(path) ?? path;
 				    filename = Path.GetFileName(path) ?? "*";
 			    }
-			    foreach (var file in Directory.GetFiles(directory, filename, SearchOption.AllDirectories))
+			    foreach (var file in Directory.GetFiles(directory, filename, searchOption))
```

Comment:
>var [](start = 16, length = 3)

make 'string' type explicit. standard says don't use var to capture assignment from any function call , as type isn't explicit in code

Reply:
Done.

---
In reply to: [141649329](https://github.com/Microsoft/sarif-sdk/pull/627#discussion_r141649329) [](ancestors = 141649329)